tests/integration/targets/vmware_rest_appliance/tasks/appliance_health.yml

```diff
@@ -0,0 +1,157 @@
+---
+- name: Get the database health status
+  vmware.vmware_rest.appliance_health_database_info:
+    vcenter_validate_certs: "{{ VMWARE_VALIDATE_CERTS }}"
+    vcenter_hostname: "{{ VMWARE_HOST }}"
```

Comment:
if credentials passed as env var, it should work without additional assignment. No need specify hostname, username and password.

Reply:
So, we need to fix it and pass it as environment variables. Do you want to open a task for it? Or add it in my PR as well?